entities.mjs

```diff
@@ -0,0 +1,882 @@
+      search_hint_type: 2,
+    },
+    annotations: {
+      edit_type: 144,
+    },
+    cache: {
+      id: 30,
+    },
+    collections: true,
+    disambiguation: true,
+    edit_table: true,
+    last_updated_column: true,
+    mbid: {
+      indexable: true,
+      multiple: true,
+      relatable: 'overview',
+    },
+    merging: true,
+    model: 'Series',
+    plural: 'series',
+    plural_url: 'series',
+    removal: {
+      automatic: ({}: AutomaticRemovalPropsT),
+    },
+    report_filter: true,
+    subscriptions: {
+      deleted: true,
+      entity: true,
+    },
+    tags: true,
+    type: {
+      simple: true,
+    },
+    url: 'series',
+  },
+  series_alias_type: {
+    cache: {
+      id: 48,
+    },
+    model: 'SeriesAliasType',
+    table: 'series_alias_type',
+  },
+  series_ordering_type: {
+    cache: {
+      id: 31,
+    },
+    model: 'SeriesOrderingType',
+  },
+  series_type: {
+    cache: {
+      id: 32,
+    },
+    model: 'SeriesType',
+  },
+  tag: {
+    cache: {
+      id: 33,
+    },
+    model: 'Tag',
+  },
+  track: {
+    artist_credits: 1,
+    last_updated_column: true,
+    mbid: {
+      multiple: true,
+    },
+    model: 'Track',
+    plural: 'tracks',
+    plural_url: 'tracks',
+    url: 'track',
+  },
+  url: {
+    edit_table: true,
+    last_updated_column: true,
+    mbid: {
+      multiple: true,
+      no_details: true,
+      relatable: 'overview',
+    },
+    model: 'URL',
+    plural: 'urls',
+    plural_url: 'urls',
+    removal: {
+      automatic: ({}: AutomaticRemovalPropsT),
+    },
+    url: 'url',
+  },
+  work: {
+    aliases: {
+      add_edit_type: 46,
+      delete_edit_type: 47,
+      edit_edit_type: 48,
+      search_hint_type: 2,
+    },
+    annotations: {
+      edit_type: 45,
+    },
+    cache: {
+      id: 34,
+    },
+    collections: true,
+    disambiguation: true,
+    edit_table: true,
+    last_updated_column: true,
+    mbid: {
+      indexable: true,
+      multiple: true,
+      relatable: 'overview',
+    },
+    merging: true,
+    meta_table: true,
+    model: 'Work',
+    plural: 'works',
+    plural_url: 'works',
+    ratings: true,
+    removal: {
+      automatic: ({}: AutomaticRemovalPropsT),
+    },
+    report_filter: true,
+    reviews: true,
+    series: true,
+    sitemaps_lastmod_table: true,
+    tags: true,
+    type: {
+      simple: true,
+    },
+    url: 'work',
+  },
+  work_alias_type: {
+    cache: {
+      id: 49,
+    },
+    model: 'WorkAliasType',
+    table: 'work_alias_type',
+  },
+  work_attribute: {
+    cache: {
+      id: 35,
+    },
+    model: 'WorkAttribute',
+  },
+  work_attribute_type: {
+    cache: {
+      id: 36,
+    },
+    model: 'WorkAttributeType',
+  },
+  work_attribute_type_allowed_value: {
+    cache: {
+      id: 37,
+    },
+    model: 'WorkAttributeTypeAllowedValue',
+  },
+  work_type: {
+    cache: {
+      id: 38,
+    },
+    model: 'WorkType',
+  },
+};
+
+deepFreeze(ENTITIES);
```

Comment:
I understand that this effectively makes ENTITIES immutable to the point where flow does not require a declared type because it's just literally what is there? If so, this might allow us to `flow strict` or `flow strict-local` some of the files which were accessing it, FWIW.

Reply:
Not quite.  This just prevents `ENTITIES` from being modified at runtime (an error will be thrown).  Flow doesn't know what this means, though, so the actual type will remain writable.  This module is marked as `@flow strict` either way, so it doesn't matter if the type is writable or read-only.

Reply:
Made it so the exported type is now read-only too.